transformers/integration_tests/integration_tests_suite_test.go

```diff
@@ -25,7 +25,7 @@ var (
 
 var _ = BeforeSuite(func() {
 	testConfig := viper.New()
-	testConfig.SetConfigName("testing")
+	testConfig.SetConfigName("docker")
 	testConfig.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
 	err := testConfig.ReadInConfig()
```

Comment:
How does the DB get set for the integration tests?

Reply:
Great question - we're calling `test_config.SetTestConfig` in the 0.go file, which sets the db info. I think we must be loading 0.go somewhere in setting up the integration test suite. I'll try to see if I can call `test_config.SetTestConfig` more explicitly so that it's clearer.

Reply:
Ohhhh, so on line 47, we're calling `db = test_config.NewTestDB(blockChain.Node())` which initializes the test_config.go file, which sets the test config via `SetTestConfig`. I'll plan to add a comment explaining this.